packages/libs/src/sdk/api/albums/AlbumsApi.ts

```diff
@@ -45,6 +45,8 @@ import {
   UploadAlbumRequest
 } from './types'
 
+const PURCHASE_MINIMUM_MICRO_LAMPORTS = 150000
```

Comment:
nit 150_000

Reply:
I feel like we've been burned by that before on React Native... going to keep as-is

Reply:
prod may have 160_000 for priority fees right now then 😱 let me test

Reply:
it's working ok right now thank god.

Reply:
Ok cool, I'll change this then. Maybe it's been resolved by changing our target from ES5? I remember there being some issue here

Reply:
up to you! maybe it doesn't work react but works in SDK?